src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -361,7 +361,8 @@ public synchronized void unlockNames(@Nullable List<String> resourceNamesToUnLoc
 		// remove context from queue and process it
 		requiredResourceForNextContext = checkResourcesAvailability(nextContext.getResources(), null, resourceNamesToUnLock);
 		this.queuedContexts.remove(nextContext);
-			
+		LOGGER.log(Level.INFO, "Queued Contexts " + this.queuedContexts);
```

Comment:
`Level.FINE` would be more appropriate. And a check on `LOGGER.isLoggable(Level.FINE)` before calling the log.

Reply:
I've add these logging changes